kbmadm/kbmadm.c

```diff
@@ -1136,3 +1173,28 @@ assert_libzfs(void)
 
 	return (errfno("libzfs_init", errno, "cannot initialize libzfs"));
 }
+
+static struct {
+	const char *str;
+	bunyan_level_t lvl;
+} levels[] = {
+	{ "trace", BUNYAN_L_TRACE },
+	{ "debug", BUNYAN_L_DEBUG },
+	{ "info", BUNYAN_L_INFO },
+	{ "warn", BUNYAN_L_WARN },
+	{ "error", BUNYAN_L_ERROR },
+	{ "fatal", BUNYAN_L_FATAL }
+};
```

Comment:
Any reason not to have this be declared static within `parse_log_level`?

Reply:
Just preference -- not a lot of stuff (at least in illumos-gate) seems to do it that way.

Reply:
confirmed.